src/parser/monk/mistweaver/modules/spells/Vivify.js

```diff
@@ -31,6 +36,12 @@ class Vivify extends Analyzer {
 
   on_byPlayer_heal(event) {
     const spellId = event.ability.guid;
+
+    // todo .hasBuff should use the sourceID so that things work with multiple mistweavers in the group
```

Comment:
i don't see `hasBuff` being called here. is this todo out of date? or does this only work correctly with a single MW in the group?

Reply:
Todo is out of date. It has been removed. I have also tested with a log with two MWs just to make sure.